src/NzbDrone.Core/Extras/ExtraService.cs

```diff
@@ -66,15 +66,13 @@ public void ImportExtraFiles(LocalMovie localMovie, MovieFile movieFile, bool is
             var sourcePath = localMovie.Path;
             var sourceFolder = _diskProvider.GetParentFolder(sourcePath);
             var sourceFileName = Path.GetFileNameWithoutExtension(sourcePath);
-            var files = _diskProvider.GetFiles(sourceFolder, SearchOption.TopDirectoryOnly);
+            var files = _diskProvider.GetFiles(sourceFolder, SearchOption.AllDirectories).OrderByDescending(d => d).ToArray();
```

Comment:
Search in all directories instead of top level only. Need to order by descending so that subs with higher detail ie. HI will be added first and get a lower prefix allowing media players to choose those first.